system/lib/compiler-rt/stack_limits.S

```diff
@@ -15,14 +15,17 @@
 #endif
 
 .globaltype __stack_pointer, PTR
+.globl __stack_pointer
 
 .section .globals,"",@
 
 # TODO(sbc): It would be nice if these we initialized directly
 # using PTR.const rather than using the `emscripten_stack_init`
 .globaltype __stack_end, PTR
 __stack_end:
+.globl __stack_end
 .globaltype __stack_base, PTR
+.globl __stack_base
```

Comment:
I don't love that these are now globals symbols...I wish we could find a way to avoid the need for divergence here between wasm workers and pthreads, but for now this seems reasonable.